pkg/scheduler/scheduler_test.go

```diff
@@ -425,3 +426,78 @@ func getPodFromPriorityQueue(queue *internalqueue.PriorityQueue, pod *v1.Pod) *v
 
 	return nil
 }
+
+func TestInitPluginsWithIndexers(t *testing.T) {
+	name := "AddIndexer"
+	entrypoint := func(obj runtime.Object, handle framework.Handle) (framework.Plugin, error) {
+		podInformer := handle.SharedInformerFactory().Core().V1().Pods()
+		podInformer.Informer().GetIndexer().AddIndexers(cache.Indexers{
+			"nodeName": indexByPodNodeName,
```

Comment:
Can you handle the returned error, and add a sub-test to demonstrate two plugins adding indexers with the same key (but with different function body)?

Reply:
sure thing